CONTRIBUTION.md

```diff
@@ -0,0 +1,184 @@
+  * [Java Styleguide](#java-styleguide)
+  * [Documentation Styleguide](#documentation-styleguide)
+
+## Code of Conduct
+
+This project is hosted by the Eclipse foundation and hence adheres to the 
+[Eclipse Code of Conduct](https://wiki.eclipse.org/Eclipse_Code_of_Conduct) that is based on the
+[Contributor Covenant](https://www.contributor-covenant.org/).
+
+## What should I know before I get started?
+
+### Eclipse Contributor Agreement
+
+Before your contribution can be accepted by the project team contributors must
+electronically sign the Eclipse Contributor Agreement (ECA).
+
+* http://www.eclipse.org/legal/ECA.php
+
+Commits that are provided by non-committers must have a Signed-off-by field in
+the footer indicating that the author is aware of the terms by which the
+contribution has been provided to the project.
+To achieve this, sign your commits by adding `-s` to your `git commit` command. 
+The non-committer must additionally have an Eclipse Foundation account and 
+must have a signed Eclipse Contributor Agreement (ECA) on file.
+Your git signature must match your Eclipse Foundation account information to be accepted.
+If your signature was accepted is visible by checking the status check on your pull request. 
+
+For more information, please see the Eclipse Committer Handbook:
+https://www.eclipse.org/projects/handbook/#resources-commit
+
+> **Note**: You can submit issues without signing the ECA. 
+
+### Contact
+
+Contact the project developers via the project's "dev" list.
+
+* https://dev.eclipse.org/mailman/listinfo/antenna-dev
+
+## How Can I Contribute?
+
+### Before Submitting An Issue
+
+* **Check the [troubleshooting documentation](https://github.com/eclipse/antenna/blob/master/antenna-documentation/src/site/markdown/troubleshooting.md.vm).** 
+You might be able to find the cause of the problem and fix things yourself. Most importantly, check if you can reproduce the problem [in the current version of Antenna](https://github.com/eclipse/antenna/blob/master). 
+It is possible that the bug has already been fixed. 
+* **Check the [documentation](https://github.com/eclipse/antenna/tree/master/antenna-documentation/src/site/markdown)**.
+Your issue might already be mentioned there. 
+* **Perform a [cursory search](https://github.com/eclipse/antenna/issues?utf8=%E2%9C%93&q=is%3Aissue)** to see if the problem has already been reported. 
+If it has *and the issue is still open*, add a comment to the existing issue instead of opening a new one.
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you want to report, open a new issue and include a link to the original issue in the body of your new one.
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report.
+Before creating bug reports, please check [the "before-submitting-a-bug" list](#before-submitting-a-bug-report) as you might find out that you don't need to create one. 
```

Comment:
This link is broken because the section is called "...issue..." (not "...bug...").